### ISSUE TITLE
Update of Database reference

### ISSUE THREAD

**Issue by MarcelBag** [firebase/geofire-java#151]
Hey guys!
As declaration of Firebase Database have been updated it seem like GeoFire doesn't work also am trying to get the vaalue of the GeoFire but it doesn't work what should I do please?

```java
public class login extends AppCompatActivity{
private DatabaseReference mDatabase
 GeoFire geoFire;
....
public void onLocationChanged(Location location){
      mDatabase = FirebaseDatabase.getInstance().getReference("path/...");
 geoFire = new GeoFire(mDatabase); 
        geoFire.setLocation(userId, new GeoLocation(location.getLatitude(), location.getLongitude()));
}
```

need help as am trying to save the latitude and longitude of the user in Firebase database using GeoFire.

**Comment by samtstern** [comment]
@MarcelBag are you getting some sort of error?  Can you explain what's happening and what you would expect to happen?

**Comment by MarcelBag** [comment]
Hello, Am not getting any error!
Am having a real time firebase database where I have authentification and
Database!
Where I can register a user and login! when I register a user it send
directly the data to the authentification!
and when I login in, it take the UserId of the login and save it into the
realtime database which is working correctly!
My expectation is this; I want to save in the database the user location,
mean the Latitude and Longitude of the user who as been login in
Unfortunately the code doesn't sort any kind of error and it doesn't send
the location into the database.

**Comment by samtstern** [comment]
@MarcelBag it's possible this issue was fixed by the `3.0.0` release which separates `geofire-java` and `geofire-android` fully.  Give it a try and let me know (if you still have this issue)!

**Comment by fransoakusi** [comment]
Facing same problem here with the same above code no error but my expectation of saving current location longitude and latitude into the firebase not happening.  How do i go by this. Thanks
